source/server/BUILD

```diff
@@ -438,6 +438,7 @@ envoy_cc_library(
         "@envoy_api//envoy/extensions/filters/listener/proxy_protocol/v3:pkg_cc_proto",
     ] + envoy_select_enable_http3([
         "//source/common/quic:active_quic_listener_lib",
+        "//source/common/quic:client_connection_factory_lib",
```

Comment:
I'd like this removed from quic protocol test too in the hopes of future-proofing but I can do in one of my many quic PRS.